ansible/roles/postgresql/tasks/main.yml

```diff
@@ -90,12 +90,16 @@
   tags: postgresql_conf
   notify: Reload postgres
 
-- name: start postgresql
+- name: Restart postgresql
   become: true
-  service: name=postgresql state=started args="{{ postgresql_version }}"
+  service: name=postgresql state=restarted args="{{ postgresql_version }}"
   tags: after-reboot
   register: start_postgresql
 
+- name: Reload postgresql
+  become: true
+  service: name=postgresql state=reloaded args="{{ postgresql_version }}"
```

Comment:
why would you need a reload immediately after a restart?

Reply:
I don't know, but my distinct impression after some testing (and the reason I added this) is that the following commands (`Create PostgreSQL users` specifically) breaks without this. I may have conflated this change with changing `started` to `restarted`, so let me just check again without this.

Reply:
OK, that's definitely what happened; made two changes at once without realizing it and took the wrong one as the operative change. Thanks for the sharp eyes!